client/src/views/calendar/calendar.module.scss

```
@@ -0,0 +1,106 @@
+@import '../../core/styles/color.scss';
+
+.calendar-page {
+  position: absolute;
+  top: 172px;
+  width: 100%;
+}
```

```suggestion

  padding-bottom: 5rem;
}
```

꼭 여기가 아니라 global 하게 main에 하단 여유가 5줄 정도 있으면 좋겠습니다.
화면 세로가 짧아서 스크롤 생기면 여유 없이 딱 붙어 있어서 보기 좋아보이지 않아요.